h5p-editor-bildetema-words-topic-image/src/styles/topicImageEditorColors.module.scss

```diff
@@ -0,0 +1,36 @@
+@use "../../../common/abstracts/" as *;
+.blueMagenta {
+  fill: $topicImageBlueMagenta !important;
```

Comment:
Would use the 0.25 opacity variable here (so the user sees how the result will look), and a higher opacity for :hover perhaps

Reply:
Could also consider to use the stroke color here also. The white stroke will not be visible on light images though, but be more visible on the dark images where the black stroke will not be as visible.